docs/_includes/chips-combobox.html

```diff
@@ -0,0 +1,396 @@
+<div id="chips-combobox">
+    {% include section-header.html name="chips-combobox" version=page.versions.chips-combobox %}
+
+    <p>Chips Combobox are creators and managers of <span class="highlight">chip</span> components. They allow user search of existing chip options, additions, and removals of chips. The search/selection follows the pattern of the combobox pattern.</p>
+
+    <h3>Empty Chips Combobox</h3>
+    <p>This is the simplest type of implementation with an empty chips combobox component usually encountered on initial information entry.</p>
+
+    <p><strong>NOTE</strong>: For empty Chips Combobox implementations, to avoid layout issues, make sure <span class="highlight">.chips-combobox__items</span> does NOT have any whitespace.</p>
+
+    <div class="demo">
+        <div class="demo__inner">
+            <div class="chips-combobox">
```

Comment:
One thing I notice different from textbox and combobox is that they default to span tag (for easy inline/side-by-side layout,  just like a regular textbox) and here we are adopting div/block level layout. Should we consider following same approach as textbox & combobox, or are their fundamental differences here?

Reply:
Technically, I suppose it's possible for `chips-combobox` to be displayed inline on the same line as other form elements, but would we ever have `chips-combobox` components side by side with other form elements or each other? I don't personally see happening since `chips-combobox` is much larger than the typical form element. 

It's the main reason I used a div here. That said, if we want the flexibility anyway, I can go ahead and use spans instead.

Reply:
I think it would be likely to be placed alongside others on desktop, especially menu-buttons. However, I think the fact that this is more of a composite component (due to it containing button chips) could lend weight to it operating differently from those other two. Same goes for the decision on 100% width really.  If it's a simple change to make it inline-block by default then maybe make the change for consistency sake with the other form fields we have built. Otherwise, maybe not a big deal if it stays the same (it just means devs will need additional css (and possibly an override) if they want things to line up side by side with it)

Reply:
Ok, I'll go ahead and make the change.